bot/exts/evergreen/profile_pic_modification/pfp_modify.py

```diff
@@ -0,0 +1,270 @@
+        This has a maximum of 512px and defaults to a 64px border.
+        The full image is 1024x1024.
+        """
+        option = option.lower()
+        pixels = max(0, min(512, pixels))
+        flag = self.GENDER_OPTIONS.get(option)
+        if flag is None:
+            await ctx.send("I don't have that flag!")
+            return
+
+        async with ctx.typing():
+            image_bytes = await ctx.author.avatar_url.read()
+            await self.send_pride_image(ctx, image_bytes, pixels, flag, option)
+
+    @prideavatar.command()
+    async def image(self, ctx: commands.Context, url: str, option: str = "lgbt", pixels: int = 64) -> None:
+        """
+        This surrounds the image specified by the URL with a border of a specified LGBT flag.
+
+        This defaults to the LGBT rainbow flag if none is given.
+        The amount of pixels can be given which determines the thickness of the flag border.
+        This has a maximum of 512px and defaults to a 64px border.
+        The full image is 1024x1024.
+        """
+        option = option.lower()
+        pixels = max(0, min(512, pixels))
+        flag = self.GENDER_OPTIONS.get(option)
+        if flag is None:
+            await ctx.send("I don't have that flag!")
+            return
+
+        async with ctx.typing():
+            async with self.bot.http_session as session:
+                try:
+                    response = await session.get(url)
+                except client_exceptions.ClientConnectorError:
+                    raise BadArgument("Cannot connect to provided URL!")
+                except client_exceptions.InvalidURL:
+                    raise BadArgument("Invalid URL!")
+
+                if response.status != 200:
+                    await ctx.send("Bad response from provided URL!")
+                    return
+
+                image_bytes = await response.read()
+                await self.send_pride_image(ctx, image_bytes, pixels, flag, option)
+
+    @prideavatar.command()
+    async def flags(self, ctx: commands.Context) -> None:
+        """This lists the flags that can be used with the prideavatar command."""
+        choices = sorted(set(self.GENDER_OPTIONS.values()))
+        options = "• " + "\n• ".join(choices)
+        embed = discord.Embed(
+            title="I have the following flags:",
+            description=options,
+            colour=Colours.soft_red
+        )
+        await ctx.send(embed=embed)
+
+    @pfp_modify.command(
+        name='spookyavatar',
+        aliases=('savatar', 'spookify'),
+        root_aliases=('spookyavatar', 'spookify', 'savatar'),
+        brief='Spookify an user\'s avatar.'
+    )
+    async def spooky_avatar(self, ctx: commands.Context, user: discord.Member = None) -> None:
+        """A command to print the user's spookified avatar."""
+        if user is None:
+            user = ctx.message.author
+
+        async with ctx.typing():
+            image_bytes = await ctx.author.avatar_url.read()
```

Comment:
The command takes in `user` as argument but doesn't use it while converting the image into bytes.
```suggestion
            image_bytes = await user.avatar_url.read()
```